pom.xml

```diff
@@ -466,7 +466,7 @@
     <version.jfree>1.0.9</version.jfree>
     <version.com.h2database>1.4.195</version.com.h2database>
     <version.org.apache.tomcat>9.0.7</version.org.apache.tomcat>
-    <version.org.apache.ant>1.8.2</version.org.apache.ant>
+    <version.org.apache.ant>1.10.9</version.org.apache.ant>
```

Comment:
We don't have this change on master, I wonder why we didn't get an alert for it?

Reply:
Maybe because the resolution mechanism is not clever enough and is not capable to manage the version in property.